site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/service/Site.java

```diff
@@ -3,19 +3,20 @@
 /**
  * String constants
  */
-public enum Services {
+public enum Site {
```

Comment:
Changing enum name from a generic 'Services' to Site - this enum is for Site messages so it seemed correct to rename.

Reply:
Agree - maybe SiteMessages in fact?

Reply:
as below